pom.xml

```diff
@@ -336,5 +340,22 @@
               </plugins>
           </build>
       </profile>
+      <profile>
+          <id>add-jaxb-api-pluginjdk-11</id>
+          <activation>
+              <property>
+                  <name>java.specification.version</name>
+                  <value>11</value>
```

Comment:
So this would be activated when the Maven JDK is 11. Awkward but OK. Is there some harm in adding `jaxb` to the `test` classpath in all cases?

Reply:
There _should_ not be https://github.com/jenkinsci/artifact-manager-s3-plugin/pull/87#issuecomment-477148738

Reply:
Though with JAXB plugin 2.3.0.1, it's impossible without bumping the baseline to 2.163+.

Reply:
I see.